.github/workflows/e2e.yaml

```diff
@@ -145,9 +145,6 @@ jobs:
 
     steps:
 
-      - name: Install curl on the machine
```

Comment:
How are you installing `clusteradm` without `curl`?
https://github.com/kubernetes-sigs/kernel-module-management/pull/401/files#diff-448746927799f21be03f629be184a58511d510c0496399d1907b9e1954be9cc1R6

Reply:
It's already installed on the node.

Reply:
This is wired, before adding this installation, I was getting `curl: command not found` during the installation of `clusteradm`....

Reply:
Could be. If it was not there I would anyway switch to `wget` to avoid installing anything.